### PR TITLE
[CHF-571] Added Health Check Implementation for EcoNet Vent.

### PR DESCRIPTION
**Pull request by skt123**
@jackchi @ShunmugaSundar 
Added Health check for EcoNet Vent with checkInterval of 32 mins.
Please review and merge.

### CODE REVIEW

File: devicetypes/smartthings/econet-vent.src/econet-vent.groovy [hunk 1]
```diff
@@ -53,7 +55,7 @@ metadata {
 
 	tiles {
 		standardTile("switch", "device.switch", width: 2, height: 2, canChangeIcon: true) {
-			state "on", action:"switch.off", icon:"st.vents.vent-open-text", backgroundColor:"#00a0dc"
+			state "on", action:"switch.off", icon:"st.vents.vent-open-text", backgroundColor:"#53a7c0"
```
Comment (jackchi) [inline devicetypes/smartthings/econet-vent.src/econet-vent.groovy]: Rebase from master again. This value should be `00adc`.

**Review by jackchi — CHANGES_REQUESTED** [review]
Other than color value, it's good.

**Comment by skt123** [comment]
@jackchi Removed color value change.
Please check.

**Review by jackchi — APPROVED** [review]
LGTM 👍

**Pull request merged**